tdigest/include/tdigest_impl.hpp

```diff
@@ -597,6 +597,24 @@ bool tdigest<T, A>::is_single_value() const {
   return get_total_weight() == 1;
 }
 
+template<typename T, typename A>
```

Comment:
why? what does this do differently compared to the implicit one?

Reply:
Maybe my understanding is wrong. I assumed this change is required to pass on the allocators to the members.